src/main.rs

```diff
@@ -138,8 +138,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
 
     println!("Loading recipes");
-    let api_recipes: Vec<api::Recipe> =
-        request::ensure_paginated_cache(&api_recipes_path, "recipes").await?;
+    let api_recipes = {
+        let mut api_recipes: Vec<api::Recipe> = request::ensure_paginated_cache(
+            &api_recipes_path, "recipes"
+        ).await?;
+        // Blacklist a couple of known bad recipes
+        for id in vec![2812, 2825] {
+            if let Some(pos) = api_recipes.iter().position(|r| r.id == id) {
+                api_recipes.remove(pos);
+            }
```

Comment:
This might be easier to do once `recipes_map` is created. You should be able to remove the recipes there with:
```rust
for id in &RECIPE_BACKLIST_IDS {
    recipes_map.remove_id(id);
}
```

Reply:
I wanted to make sure I removed them _before_ the map was created, so any valid recipes for the same item were still craftable. You'll note I blacklisted the recipe instead of the output.

Reply:
You can verify it's working via:
```
$ target/debug/gw2-arbitrage 24748
<snip>
Fetching https://api.guildwars2.com/v2/commerce/listings?ids=24301,24748,89103,89140,89271
Item is not profitable to craft
```
You'll note the item ids it's looking up are the ones for the valid recipe.

Reply:
Ah right, I forgot `recipes_map` is keyed on the item id.